pkg/interface/src/views/apps/launch/app.js

```diff
@@ -39,19 +42,18 @@ export default function LaunchApp(props) {
           pt={0}
         >
           <Tile
-            bg="transparent"
-            color="green"
+            bg="white"
+            color="scales.black20"
             to="/~landscape/home"
             p={0}
           >
-            <Box p={2} height='100%' width='100%' bg='green'>
+            <Box p={2} height='100%' width='100%' bg='scales.black20'>
               <Row alignItems='center'>
                 <Icon
-                  color="white"
-                  // fill="rgba(0,0,0,0)"
-                  icon="Boot"
+                  color="black"
+                  icon="Mail"
```

Comment:
Look through Leap's OmniboxResults + Landscape's GroupSwitcher to adjust DM + Draft's icon "Boot" to match this one, "Mail".